app/src/main/java/com/heig/atmanager/Goal.java

```diff
@@ -30,13 +30,15 @@
     
     private String unit;
     private int quantity;
+    private int intervalNumber;
```

Comment:
Not sure I understand the "intervalNumber" variable, what's its purpose ?

Reply:
It's for how much days/week/... for the interval. It's allow the user to specifie a goal that repeate every 2 days for example